src/main/resources/templates/add.html

```diff
@@ -1,43 +1,35 @@
 <!DOCTYPE html>
-<html xmlns:th = "http://www.thymeleaf.org">
+<html lang="en" xmlns:th = "http://www.thymeleaf.org">
 <head>
+  <title>Investment Tracker</title>
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1">
-  <title>Investment Tracker</title>
   <link rel="stylesheet" href="style.css">
   <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.2.2/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-Zenh87qX5JnK2Jl0vWa8Ck2rdkQ2Bzep5IDxbcnCeuOxjzrPF/et3URy9Bv1WTRi" crossorigin="anonymous">
+  <script src="script.js"></script>
   <script src="https://code.jquery.com/jquery-1.12.4.js"></script>
   <script src="https://code.jquery.com/ui/1.12.1/jquery-ui.js"></script>
   <link rel="stylesheet" href="//code.jquery.com/ui/1.12.1/themes/base/jquery-ui.css">
-  <script type="text/javascript">
-    $(function() {
-      $("#company_Symbol").autocomplete({
-        source: "dataAutoComplete",
-        minLength: 1
-      });
-    });
-  </script>
 </head>
 <body>
 <div th:replace="fragments/topnav :: navbar">
-
 </div>
 <h2>Open</h2>
 <p th:text="${investment}"></p>
 <form action="#" th:action="@{/openInvestment}" th:object="${investment}" method="GET">
   <div class="input-group input-group-sm mb-3">
     <span class="input-group-text" id="symbol">Symbol</span>
-    <input id="company_Symbol" type="text" class="form-control" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-sm" name="symbol" th:field="*{symbol}">
+    <input id="company_Symbol" type="text" class="form-control" aria-label="Sizing example input" aria-describedby="symbol" name="symbol" th:field="*{symbol}">
   </div>
   <div class="input-group input-group-sm mb-3">
     <span class="input-group-text" id="shares">Shares</span>
-    <input type="text" class="form-control" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-sm" name="shares" th:field="*{shares}">
+    <input type="text" class="form-control" aria-label="Sizing example input" aria-describedby="shares" name="shares" th:field="*{shares}">
   </div>
   <div class="input-group input-group-sm mb-3">
     <span class="input-group-text" id="priceOpened">Price Opened</span>
-    <input type="text" class="form-control" aria-label="Sizing example input" aria-describedby="inputGroup-sizing-sm" name="priceOpened" th:field="*{priceOpened}">
+    <input type="text" class="form-control" aria-label="Sizing example input" aria-describedby="priceOpened" name="priceOpened" th:field="*{priceOpened}">
   </div>
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-describedby

aria-describedby should be assigned to an associated ID. It was being assigned to a class. I changed them to the correct IDs which resolved the errors that validation was throwing.